modules/govuk_postgresql/templates/etc/cron.daily/autopostgresqlbackup.erb

```diff
@@ -344,6 +344,16 @@ dbdump () {
   touch $2
   chmod 600 $2
   for db in $1 ; do
+    <% if @environment != 'integration' %>
```

Comment:
Your reading is, but the code is not. I've now fixed it such that it only includes this special case on integration.